### PR TITLE
Add Procfile

### PR DESCRIPTION
**Pull request by Mrjaco12**
Why:

* Heroku will want it to run our app

This change addresses the need by:

* Adding a simple default procfile
* Adding a recipe for installing it

### CODE REVIEW

**Comment by samcdavid** [comment]
Does this cover installing `foreman` in the development group? I don't think that we have required that in the past. However, it wouldn't be a bad idea.

**Comment by Mrjaco12** [comment]
It does not. We need this config to tell heroku how to run the app. We aren't planning on having Foreman run locally by default at this point.

File: recipes/foreman.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+REPO = "https://raw.githubusercontent.com/spartansystems/spartan-composer-recipes/master/files/".freeze
+
+stage_one do
+  copy_from_repo "Procfile", repo: REPO
+end
+__END__
+
+name: foreman
+description: "Add the default Foreman Procfile"
```
Comment (squaresurf) [inline recipes/foreman.rb]: These lines are probably why @samcdavid thought this was for Foreman. Should we change them to Heroku? Not to mention the recipe filename.

**Review by squaresurf — CHANGES_REQUESTED** [review]

File: recipes/foreman.rb (hunk 1 above)
Comment (Mrjaco12) [reply recipes/foreman.rb]: Any recommendation on a new name? Procfile? It is for Foreman. Just not local Foreman lol.

**Review by squaresurf — APPROVED** [review]

**Comment by Mrjaco12** [comment]
[Rails Auth Flow](https://trello.com/c/9M0vleig/643-rails-auth-flow)

**Pull request merged**